src/utils/api.ts

```diff
@@ -3,14 +3,16 @@ import axios from 'axios';
 import Options from './options';
 
 const options = new Options();
+const baseURL = 'https://wakatime.com';
```

Comment:
Because it's in the modeule `API` let's make it ending with `/api`.

Reply:
done